main_test.go

```diff
@@ -540,6 +563,41 @@ v6 = 42
 v7 = 666.666
 k1 = 1
 k2 = 2
+New values
+v1 = test1
+v2 = 90
+v3 = 1111.1111
+v4 = test2
+v5 = 50
+v6 = 50
+v7 = 1111.1111
+k1 = 123
+k2 = 456
+`),
+	})
+
+	testPkgWithCFFI(t, pkg{
+		path: "_examples/vars",
+		want: []byte(`Initial values
+v1 = v1
```

Comment:
and for one `Get/Set` pair.

it's OK to have a different reference for CFFI and for CPython-2.
(if there is no issue number for the CPython-2 backend to generate docstrings for `Get/Set`s of vars, I'll add one.)